index.bs

```diff
@@ -1610,7 +1655,8 @@ Issue: handle String / Number / etc. wrapper objects specially?
 <div algorithm>
 
 To <dfn>serialize as a remote value</dfn> given a |value|, a |max depth|,
-|node details|, a |set of known objects| and a |realm|:
+a |node details|, an |ownership|, a |set of known objects|, an
```

Comment:
We don't really need |set of known objects| and |internal object remote value map|, we can just have a single map that either a) gives each object a random id when its inserted or b) sets the id to null unless the object is later referenced as part of a cycle.

Reply:
The problem is we don't have remote value at the moment we want to serialize the value. I can either:
1. Create RemoteValue before serializing each value.
2. Put `null` into `serialization internal map`
3. Use `set of known objects`.

I don't like option `2`, but option `1` seems reasonable. I'm working on fixing that.

Reply:
option `1` "Create RemoteValue before serializing each value" is implemented.